### PR TITLE
Banner prompting users to migrate to FxA (fixes #986)

### LINKED ISSUE #986: Banner to prompt user to migrate to FxA
There should be a banner that user's see if they are logged in but have not migrated to FxA yet. There should be a call to action link that sends the user to FxA with their user id in the state.

screenshot 2015-11-26 14 26 46

### PR DESCRIPTION
**Pull request by mstriemer**
fxa-migration-banner-2 mov

It's a banner with a link.

### CODE REVIEW

File: templates/impala/base.html [hunk 1]
```diff
@@ -125,6 +125,27 @@
...
+                <div class="site-balloon">
+                  <p>
+                    {% trans url='http://mozilla.org/firefox/?src=amo' %}
+                      To try the thousands of add-ons available here, download
+                      <a href="{{ url }}">Mozilla Firefox</a>,
+                      a fast, free way to surf the Web!
+                    {% endtrans %}
+                  </p>
+                  <a href="#" class="close">{{ _('Close') }}</a>
+                </div>
```
Comment (magopian) [inline templates/impala/base.html]: What is this above?

File: static/js/impala/global.js [hunk 2]
```diff
@@ -167,6 +171,8 @@ function initBanners(delegate) {
             z.visitor.set('seen_acr_pitch', 1);
         } else if ($parent.is('#appruntime-pitch')) {
             z.visitor.set('seen_appruntime_pitch', 1);
+        } else if ($parent.is('#fxa-migration-banner')) {
+            z.visitor.set('seen_fxa_migration_banner', 1);
```
Comment (magopian) [inline static/js/impala/global.js]: Does that mean they'll never ever see it again? Is that an issue?

**Comment by magopian** [comment]
Looks good ;)

r+wc

File: templates/impala/base.html (hunk 1 above)
Comment (mstriemer) [inline templates/impala/base.html]: I don't know how that got in there. I'll pull it out.

File: static/js/impala/global.js (hunk 2 above)
Comment (mstriemer) [inline static/js/impala/global.js]: They won't see it until `localStorage` gets cleared. I'll put it in `sessionStorage` instead.

**Comment by mstriemer** [comment]
Now more annoying!

fxa-migration-banner-3 mov

**Comment by magopian** [comment]
Nice, I really like those screencast you're doing ;)

**Pull request merged**